test/unit/ldp/auxiliary/RdfValidator.test.ts

```diff
@@ -38,7 +39,5 @@ describe('An RdfValidator', (): void => {
     converter.handleSafe = jest.fn().mockRejectedValue(new Error('bad data!'));
     const representation = new BasicRepresentation('data', 'content-type');
     await expect(validator.handle(representation)).rejects.toThrow('bad data!');
-    // Make sure the data on the readable has not been reset
-    expect(representation.data.readableEnded).toBe(true);
```

Comment:
We probably still want this to be true though, since otherwise we have an unused stream that is not closed. So there should be a try/catch around the conversion attempt which destroys this stream on failure (and then re-throws the error).

Reply:
Weird, it should. It might simply be an async issue so that it's waiting for an event before that boolean is set. You could change the test to check if `data.destroyed` is true instead.